src/liballoc/collections/linked_list.rs

```diff
@@ -1197,6 +1197,19 @@ impl<T: Clone> Clone for LinkedList<T> {
     fn clone(&self) -> Self {
         self.iter().cloned().collect()
     }
+
+    fn clone_from(&mut self, other: &Self) {
+        let mut iter_other = other.iter();
+        if self.len() > other.len() {
+            self.split_off(other.len());
+        }
+        for elem in self.iter_mut() {
+            elem.clone_from(iter_other.next().unwrap());
+        }
```

Comment:
Could this be written using `zip`, just to avoid the unnecessary unwrap? Then we have `for (elem, elem_other) in self.iter_mut().zip(&mut iter_other) {`

Reply:
Yes, that also definitely matches the actual program logic better. Just checking to make sure I didn't mess anything up when I changed it over. Thank you!